pkg/commands/image/trigger.go

```diff
@@ -63,12 +63,13 @@ The namespace defaults to the kubernetes current-context namespace.`,
 					return err
 				}
 
-				_, err = cs.KpackClient.KpackV1alpha2().Builds(cs.Namespace).Patch(ctx, original.Name, types.MergePatchType, patch, metav1.PatchOptions{})
+				builds, err := cs.KpackClient.KpackV1alpha2().Builds(cs.Namespace).Patch(ctx, original.Name, types.MergePatchType, patch, metav1.PatchOptions{})
 				if err != nil {
 					return err
 				}
+				buildNumber := builds.Labels["image.kpack.io/buildNumber"]
```

Comment:
This should be using the const from the [kpack api](https://github.com/buildpacks-community/kpack/blob/main/pkg/apis/build/v1alpha2/image_builds.go#L17) rather than hard coding it here.